Workbooks/Traffic Analytics/Unified/ApplicationPorts.workbook

```diff
@@ -0,0 +1,955 @@
+                    },
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "FlowType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "VNet",
+                        "resultValType": "static",
+                        "resultVal": "{isVNetTopologySchemaPresent},VNet"
+                      }
+                    },
+                    {
+                      "criteriaContext": {
+                        "operator": "Default",
+                        "resultValType": "static",
+                        "resultVal": "{isNSGTopologySchemaPresent},NSG,{isVNetTopologySchemaPresent},VNet"
+                      }
+                    }
+                  ],
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                },
+                {
+                  "id": "b1715201-f711-4a92-bf97-4720b2e295c7",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "SubscriptionQueryToRun",
+                  "type": 1,
+                  "isHiddenWhenLocked": true,
+                  "criteriaData": [
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "QueryType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "true,NSG",
+                        "resultValType": "static",
+                        "resultVal": "AzureNetworkAnalytics_CL | extend timecheck = iff({timeInterval:seconds} < 86400, TimeGenerated between (datetime_add('day', -1, todatetime('{timeInterval:endISO}')) .. todatetime('{timeInterval:endISO}')), TimeGenerated between (todatetime('{timeInterval:startISO}') .. todatetime('{timeInterval:endISO}'))) | where SubType_s == 'StatusMessage' and ComponentType_s == 'Topology' | distinct Subscription_g, SubscriptionName_s | project Subscription = Subscription_g, SubscriptionName = SubscriptionName_s, selected = 1"
+                      }
+                    },
+                    {
+                      "criteriaContext": {
+                        "leftOperand": "QueryType",
+                        "operator": "==",
+                        "rightValType": "static",
+                        "rightVal": "true,VNet",
+                        "resultValType": "static",
+                        "resultVal": "NTATopologyDetails | extend timecheck = iff({timeInterval:seconds} < 86400, TimeGenerated between (datetime_add('day', -1, todatetime('{timeInterval:endISO}')) .. todatetime('{timeInterval:endISO}')), TimeGenerated between (todatetime('{timeInterval:startISO}') .. todatetime('{timeInterval:endISO}'))) | where timecheck | where SubType == 'StatusMessage' and ComponentType == 'Topology' | distinct Subscription, SubscriptionName | project Subscription, SubscriptionName, selected = 1"
```

Comment:
What are the other possible values of ComponentType in NTATopologyDetails table?

Reply:
Possible values are Flowlog/Topology.